docs/rest-api.md

```diff
@@ -29,20 +109,75 @@ The root URL for the the API is found on the same page as:
 window.Initial_State.WP_API_root;
 ```
 
-## Discovery
+## API Reference
+
+All endpoints return and accept JSON. Make sure you add the proper `content-type` header to your PUT/POST requests sending JSON objects.
+
+```
+'Content-type': 'application/json'
+```
+
+### Discovery endpoint
 
-WP-API-compatible [capabilities document](http://v2.wp-api.org/guide/discovery/) for the endpoints registered by Jetpack.
+Core REST API-compatible [capabilities document](https://developer.wordpress.org/rest-api/using-the-rest-api/discovery/) for the endpoints registered by Jetpack.
 
 `GET /wp-json/jetpack/v4`
 
-## API Reference
+### Jetpack settings
 
-All endpoints return and accept JSON. Make sure you add the proper `content-type` header to your PUT/POST requests sending JSON objects.
+We call settings to any option that a module provides.
```

Comment:
Can you please rephrase this sentence? I don't understand what you meant to say here.